.gitignore

```diff
@@ -1,7 +1,7 @@
 .php_cs.cache
 composer.phar
 composer.lock
+phpunit.xml
```

Comment:
Since what you have is `phpunit.xml.dist`, the intent should be for `phpunit.xml` to be a locally-modified copy of it... And this line doesn't match `phpunit.xml.dist`